src/SDKs/ContainerRegistry/dataplane/Microsoft.Azure.ContainerRegistry.Tests/Microsoft.Azure.ContainerRegistry.Tests.csproj

```diff
@@ -0,0 +1,28 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.test.reference.props'))" />
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.ContainerRegistry.Tests</PackageId>
+    <Description>Microsoft.Azure.ContainerRegistry tests Library</Description>
+    <AssemblyName>Microsoft.Azure.ContainerRegistry.Tests</AssemblyName>
+    <Version>1.0.0</Version>
```

Comment:
Please add `<TreatWarningsAsErrors>true</TreatWarningsAsErrors>` and fix all errors